src/interfaces/telemetryEvent.ts

```diff
@@ -0,0 +1,8 @@
+export interface TelemetryEvent {
+  extensionName: string; // name of extension
```

Comment:
the extension name should be held by the Reporter instance

Reply:
done